pytorch/mnist/requirements-remote.txt

```diff
@@ -1,3 +1,4 @@
+tensorflow==1.15.2
```

Comment:
This isn't required because it is already installed in the base image.

Reply:
AFAIK Tensorflow is not installed in the pytorch base image. I added this because simply installing tensorboard threw errors about tensorflow not being installed.